telegram/_passport/passportelementerrors.py

```diff
@@ -167,35 +168,54 @@ class PassportElementErrorFiles(PassportElementError):
         type (:obj:`str`): The section of the user's Telegram Passport which has the issue, one of
             ``"utility_bill"``, ``"bank_statement"``, ``"rental_agreement"``,
             ``"passport_registration"``, ``"temporary_registration"``.
-        file_hashes (Sequence[:obj:`str`]): List of base64-encoded file hashes.
+        file_hashes (List[:obj:`str`]): List of base64-encoded file hashes.
         message (:obj:`str`): Error message.
 
     Attributes:
         type (:obj:`str`): The section of the user's Telegram Passport which has the issue, one of
             ``"utility_bill"``, ``"bank_statement"``, ``"rental_agreement"``,
             ``"passport_registration"``, ``"temporary_registration"``.
-        file_hashes (Tuple[:obj:`str`]): List of base64-encoded file hashes.
         message (:obj:`str`): Error message.
 
     """
 
-    __slots__ = ("file_hashes",)
+    __slots__ = ("_file_hashes",)
 
     def __init__(
         self,
         type: str,
-        file_hashes: Sequence[str],
+        file_hashes: List[str],
         message: str,
         *,
         api_kwargs: Optional[JSONDict] = None,
     ):
         # Required
         super().__init__("files", type, message, api_kwargs=api_kwargs)
         with self._unfrozen():
-            self.file_hashes: Tuple[str, ...] = parse_sequence_arg(file_hashes)
+            self._file_hashes: List[str] = file_hashes
 
             self._id_attrs = (self.source, self.type, self.message, *tuple(file_hashes))
 
+    def to_dict(self, recursive: bool = True) -> JSONDict:
+        """See :meth:`telegram.TelegramObject.to_dict` for details."""
+        data = super().to_dict(recursive)
+        data["file_hashes"] = self._file_hashes
+        return data
+
+    @property
+    def file_hashes(self) -> List[str]:
+        """List of base64-encoded file hashes.
+
+        .. deprecated:: NEXT.VERSION
+            This attribute will return a tuple instead of a list in v22.
```

Comment:
```suggestion
            This attribute will return a tuple instead of a list in future versions.
```

let's try to hardcode these as seldom as possible :D if we want to be more precise, we can link to the stability policy.

same below.

Reply:
but we will remove it in v22 right? Also why is the stability policy not in linked in the README.md?

Reply:
Yes, that is the plan, I would just avoid hard-coding as much as possible. Just in case something unforseen happens :)
linking in the readme is a good idea, can go into the doc-fixes branch 👍

Reply:
I will go with "future major versions"